cnvlib/gary.py

```diff
@@ -495,3 +501,22 @@ def write(self, outfile=None):
         with ngfrills.safe_write(outfile or sys.stdout) as handle:
             self.data.to_csv(handle, index=False, sep='\t', float_format='%.6g')
 
+    def _get_gene_map(self) -> OrderedDict:
```

Comment:
Type annotations are deep in Python 3 territory; Python 2.7 needs this information in the docstring only.